fenix/app/src/main/java/org/mozilla/fenix/components/FindInPageIntegration.kt

```diff
@@ -30,19 +31,38 @@ import org.mozilla.fenix.components.FindInPageIntegration.ToolbarInfo
 class FindInPageIntegration(
     private val store: BrowserStore,
     private val sessionId: String? = null,
-    stub: ViewStub,
+    private val view: FindInPageBar,
     private val engineView: EngineView,
     private val toolbarInfo: ToolbarInfo,
-) : InflationAwareFeature(stub) {
```

Comment:
Can you also remove `InflationAwareFeature` as FIP was the only feature using it and we are not tempted to use it again? 😄

Reply:
You can also close [this issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1794676) as 'WONTFIX' if you do the above.